public/locales/en/layout/modals/add-app.json

```diff
@@ -72,6 +72,10 @@
 				"bottom":"Bottom",
 				"left":"Left"
 			}
+		},
+		"lineClampAppName":{
+			"label":"App Name Line Clamp",
+			"description":"Defines on how many lines your title should fit at it's maximum. Set 0 for unlimited."
```

Comment:
Adding a widget option for that is so unnecessary, please have it hard-coded.

Reply:
Plus if anything it should be a global setting. But it really shouldn;'t be a setting at all

Reply:
This is about leaving the option to the user to do it if they want to. the purpose is more for when the text is on the side like shown in the screenshots. Most people won't even bother with the setting and it is made so that the default values are making it behave exactly the same as before. it will only matter for those that want to use it.
If I hard code it to 2-3 then you'll have 2-3 lines in vertical too which wouldn't be the default behavior as much as I know some wouldn't want their text clamped, some like the consistency of one line. this is really just about letting people choose what they prefer instead of forcing our own choices onto them

Reply:
Yeah, sometimes we have to force the user. If I wanted to I could make every setting an option but we don't because it pollutes the config files and adds even more bug-prone features and stuff to maintain

Reply:
We've tried and make it as robust as possible to avoid bugs. I don't really understand what you mean with "polluting" the config, after all it's not something to just go into and tinker with and all those options are well constructed into it.
People want customization and custom CSS can only do so much to offer them that. It's not over the top and complicated either.
I agree that there's only so much we can offer in that aspect but we should offer as much as possible.
in that very case I think we no matter what we need to let the users choose between 1 and 2 lines, so we might as well offer the full range.

Reply:
I agree with @Tagaishi we should offer the users as much customizability as possible without needing to learn CSS.